src/dosarray_graphing_focus.sh

```diff
@@ -23,8 +36,8 @@ SUFFIX="\n\
 set yrange [0:100] \n\
 set style data lines \n\
 set title '${TITLE}' \n\
-set output '${DATA_DIR}/focus_graph.pdf' \n\
-plot '${DATA_DIR}/availability.data' using 2:3 smooth csplines title 'availability' \n\
+set output '${OUTPUT_FILE}' \n\
+plot '${INPUT_FILE}' using 1:2 smooth csplines title 'availability' \n\
```

Comment:
Not clear why the graphed data columns were changed -- from `2:3` to `1:2`. Is this change related to changing the parameters of this script?

Reply:
Will revert back this change. Was facing an error because I set DOSARRAY_NHIST_RESULT=1 instead of DOSARRAY_HIST_FOCUS=10